path.go

```diff
@@ -498,8 +498,29 @@ func splitNonEscaped(s string, sep byte) []string {
 	return append(result, s)
 }
 
+func hasPartialMatchBoundary(path string, matchedLength int) bool {
+	if matchedLength < 0 || matchedLength > len(path) {
+		return false
+	}
+	if matchedLength == len(path) {
+		return true
+	}
+	if matchedLength == 0 {
+		return false
+	}
+	if path[matchedLength-1] == slashDelimiter {
+		return true
```

Comment:
This line is not covered

Reply:
solved with [`05098eb` (#3755)](https://github.com/gofiber/fiber/pull/3755/commits/05098eb7bd8f69c76f04a0494ee85af3bdb6e30e)